tests/e2e-inst-signatures/scripts/containers_data_source.sh

```diff
@@ -6,5 +6,5 @@ exit_err() {
     exit 1
 }
 
-docker pull ubuntu > /dev/null
-docker run --rm ubuntu /usr/bin/ls > /dev/null
+docker pull ubuntu:jammy-20240627.1 > /dev/null
```

Comment:
Let's standardize this docker commands.

```suggestion
docker image pull ubuntu:jammy-20240627.1 > /dev/null
```